views/proposals.tmpl

```diff
@@ -118,12 +118,35 @@
                     <tr>
                         <td class="text-left"><a href="/proposal/{{.RefID}}">{{.Name}}</a></td>
                         <td class="text-left">{{.Username}}</td>
-                        {{with .Status}}
-                            <td class="text-left">{{toTitleCase .String}}</td>
-                        {{end}}
                         <td class="text-left">
-                            {{if .VoteStatus}}{{with .VoteStatus}}{{.ShortDesc}}{{end}}{{else}}N/A{{end}}
+                            {{if .VoteStatus}}
+                                {{range $i, $vr := .VoteResults}}
+                                    {{if eq $vr.Option.OptionID "yes"}}
+                                        {{$votesPercent :=  percentage $vr.VotesReceived $v.TotalVotes}}
+                                        {{if lt $.Tip.Height $v.Endheight}}
```

Comment:
the` Endheight` should be cast from string. I will be doing that later today.

Reply:
That is annoying. Ugh. Should def be a number already.